scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 699,
-  MAX_SIZE: 742,
+  MAX_SIZE: 750,
```

Comment:
now that our pages are in better order I wonder if it's possible to have separate bundles for them 🤔I'll have a dig around and maybe create an issue for it. Could be a fun one to look into if we can bring the initial load bundle size down

Reply:
https://github.com/bbc/simorgh/pull/5843

Reply:
Can't wait :nerd_face: